webext: manifest: csp - valid example needs object-src

hamishwillee opened the pull request:
Fixes #10836

From docs  "There are restrictions on the policy you can specify here: The policy must include at least the script-src and the object-src directives,"

This fixes an example of a valid case that omitted the object-src. The other examples which omit the key are under the "invalid examples".

rebloor commented on files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:
```diff
@@ -89,7 +89,7 @@ Keep the rest of the policy, but also require that images should be packaged wit
 Require that all types of content should be packaged with the extension:
```
Perhaps move this to be the first example

rebloor commented on files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:
````diff
@@ -89,7 +89,7 @@ Keep the rest of the policy, but also require that images should be packaged wit
 
 ```json
-"content_security_policy": "default-src 'self'"
+"content_security_policy": "default-src 'self'; object-src 'self'"
````
This change isn't correct, you can use `[default-src](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src)` alone. Please revert.

hamishwillee replied:
Awesome. Thanks.

rebloor reviewed (changes requested):
`default-src` alone is perfectly valid. Probably what we need to do is amend line 46 to clarify: 
`- The policy must include {{CSP("default-src")}} or at least the {{CSP("script-src")}} and the {{CSP("object-src")}} directives. Also, the {{CSP("script-src")}} directive must contain the keyword `'self'`.`

hamishwillee committed:
The policy may include just CSP default-src

hamishwillee commented:
Thanks @rebloor.

I've taken your suggestions (reverted the change, moved the example up to the top, added a comment that default-src is allowed on its own (used my own words)).

Ready for re-review.

rebloor reviewed (approved):
Looks good to me @hamishwillee
